codegen/src/main/kotlin/tools/samt/codegen/http/HttpTransport.kt

```diff
@@ -44,7 +45,7 @@ object HttpTransportConfigurationParser : TransportConfigurationParser {
                                 "Invalid operation config for '$operationName', expected '<method> <path> <parameters>'. A valid example: 'POST /${operationName} {parameter1, parameter2 in query}'",
                                 operationConfig
                             )
-                            return@mapNotNull null
+                            return@forEach
```

Comment:
I personally prefer using a traditional `for(...)` loop and `continue` statements instead